pubspec.yaml

```diff
@@ -37,7 +37,7 @@ dev_dependencies:
   build_cli: ^2.2.0
   json_serializable: ^6.3.1
   lints: ^2.0.0
-  test: ^1.14.3
+  test: ^1.22.2
```

Comment:
Released new test vision that allows `vm_service 10.0.0` and newer.
https://github.com/dart-lang/test/pull/1842
https://github.com/dart-lang/test/pull/1843